client/my-sites/activity/filterbar/index.jsx

```diff
@@ -52,13 +60,32 @@ export class Filterbar extends Component {
 		this.setState( { showActivityTypes: false } );
 	};
 
+	toggleIssueTypesSelector = () => {
+		this.setState( {
+			showActivityTypes: false,
+			showActivityDates: false,
+			showIssueTypes: ! this.state.showIssueTypes,
+		} );
+	};
+
+	closeIssueTypes = () => {
+		this.setState( { showIssueTypes: false } );
+	};
+
 	handleRemoveFilters = () => {
-		const { siteId, resetFilters } = this.props;
-		resetFilters( siteId );
+		const { resetFilters } = this.props;
+		resetFilters();
```

Comment:
Do we know why we used to pass the `siteId` to `resetFilters`?

Reply:
I understand that there is no `siteId` in the dashboard, but I'm concerned about other places where this component is used in the context of a single site (the Activity Log for a given site).